app/frontend/vite.config.mjs

```diff
@@ -64,8 +64,10 @@ export default defineConfig(({ command, mode }) => {
       },
       globals: true,
       environment: 'jsdom',
-      deps: {
-        inline: ['vuetify', 'i18n', 'vitest-canvas-mock'],
+      server: {
+        deps: {
+          inline: ['vuetify', 'i18n', 'vitest-canvas-mock'],
+        },
```

Comment:
The upgrade to a newer version of `vite` was producing warnings that our old configuration was in a deprecated format.